profile/ekstep/rootfs_overlay/etc/nginx/sites-enabled/opencdn_nginx

```diff
@@ -57,7 +57,7 @@ server {
 
 	location / {
                 #proxy_pass http://127.0.0.1:8008;
-				try_files $uri $uri/ @ekstep;
+				try_files $uri $uri/ /index.html;
```

Comment:
What is @ekstep defined as ? Why is this replaced with /index.html?

Reply:
this came as a copy paste thing, need to replace it with index.html because client is handling the routing in our case. For all the routes nginx must return the index file itself